### DISCUSSION TITLE
How to run SiamRPN++、DasiamRPN、SiamFC？

### DISCUSSION THREAD

**Discussion by dudiaoxifenglie** [STVIR/pysot#546] [Q&A]
Sorry, My question is so stupid. But, I didn't find the configs of these algorithms in pysot-master\experiments. Can you tell me?

**Comment by TheRealPoseidon** [comment]
hi~Do you know how to run them in this repository now?
I found another version of DasiamRPN in [https://github.com/foolwood/DaSiamRPN](https://github.com/foolwood/DaSiamRPN)
